i18n/locales/en-GB/dashboard/scripts.yml

```diff
@@ -98,6 +98,27 @@ en-GB:
           stages:
             Lists:
               name: Lists
+        dance:
+          stages:
+            Dance Party:
+              name: Fiesta Bailabel
+          title: Fiesta Bailabel
+        dance--draft:
+          stages:
+            Dance Party:
+              name: Fiesta Bailabel
+        dance-draft:
+          stages:
+            Dance Party:
+              name: Fiesta Bailabel
+        dance-low:
+          stages:
+            Dance Party:
+              name: Fiesta Bailabel
```

Comment:
This is another translation in the wrong language. This is definitely not in British English. :)

Reply:
@bethanyaconnor Good catch!  In Crowdin, a translator added this translation 5 days ago.

Reply:
I removed the translation; looks like that was the only translation the translator added, so this is probably an isolated mistake